chapter03/c03_flan.rpy

```diff
@@ -11,20 +11,20 @@ label c03_flan:
     He must have thought that I wouldn't do it.
     c "Ehehe... [Pause] Well..."
     c "Here you go! Make sure you eat all of it!"
-    With a smile, I extended the coffered the cup of flan to the stunned boy.
```

Comment:
Ah, this is just my hand writing more than what I had thought in my head. I'm okay with, "I extended the cup of flan to the stunned boy."